source/features/preview-hidden-comments.tsx

```diff
@@ -18,10 +17,11 @@ const init = (): void => {
 			continue;
 		}
 
-		const header = select(`
-			summary .timeline-comment-header-text,
-			summary .discussion-item-copy
-		`, details)!;
+		const header = select([
+			'summary .timeline-comment-header-text', // Issue and commit comments
+			'summary .discussion-item-copy',
```

Comment:
Can you find where this was used and add a comment specifying it?

Reply:
I should have commented inline

- Fix review comments on `preview-hidden-comments` - I am not sure what that second selector is. Its not clear from the PR or blame. (Tests it on #3242)
https://github.com/sindresorhus/refined-github/pull/1755/files/794b11f52d7e8e7721d3f4918ae55eb9d203b208#r254981750
[`42a7ef3` (#1755)](https://github.com/sindresorhus/refined-github/pull/1755/commits/42a7ef3412ad5edd12ec5eef35482d3bac22d14d)

Reply:
From this it appears that he referred to review comments that appear in the Conversation tab: https://github.com/sindresorhus/refined-github/pull/1755#issuecomment-462035220

So it can be dropped